plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/JavaUtils.java

```diff
@@ -92,28 +139,6 @@ public static void checkPackageName(String name) throws IllegalStateException {
      * @return <code>true</code> if the specified text is a valid package name, <code>false</code> otherwise
      */
     public static boolean isValidPackageName(String name) {
-        IStatus status = validatePackageName(name);
-        switch (status.getSeverity()) {
-            case Status.WARNING:
-            case Status.OK:
-                return true;
-            default:
-                return false;
-        }
-    }
-
-    private static IStatus validateCompilationUnitName(String name) {
-//        return JavaConventions.validateCompilationUnitName(name, JavaCore.getOption(JavaCore.COMPILER_SOURCE),
```

Comment:
May be we can keep using JavaConventions class by porting it to client?
It provides really useful methods for checking Java-specific conventions corresponding to the Java Language Specification. Also it provides informative hints why some name isn't valid.